apps/zotonic_mod_zotonic_site_management/src/zotonic_status_addsite.erl

```diff
@@ -278,21 +278,15 @@ copy_skeleton_dir(From, To, Options, Context) ->
             ok,
             Files).
 
-copy_file("SITE" ++ Ext, FromPath, ToPath, Options) when Ext == ".erl"; Ext == ".app.src" ->
-    % Replace with the site name
-    ToPath1 = filename:join([
-                    filename:dirname(ToPath),
-                    z_convert:to_list(proplists:get_value(site, Options)) ++ Ext
-                ]),
-    case filelib:is_file(ToPath1) of
+copy_file("SITE" ++ Ext = Filename, FromPath, ToPath, Options) when Ext == "_sup.erl"; Ext == "_app.erl" ->
+    case proplists:get_value(supervisor, Options) of
         true ->
-            ok;
+            copy_site_file(Filename, FromPath, ToPath, Options);
         false ->
-            % Replace the module name, write the site Erlang module.
-            {ok, SiteErl} = file:read_file(FromPath),
-            Outfile = replace_tags(SiteErl, Options),
-            file:write_file(ToPath1, Outfile)
+            ok
     end;
+copy_file("SITE" ++ Ext = Filename, FromPath, ToPath, Options) when Ext == ".erl"; Ext == ".app.src"; Ext == "_sup.erl"; Ext == "_app.erl" ->
```

Comment:
```suggestion
copy_file("SITE" ++ Ext = Filename, FromPath, ToPath, Options) ->
```

Reply:
These guards are a must because only erlang files have tags to replace.
In [this function](https://github.com/zotonic/zotonic/blob/35143344ce2eac224e8ce1d91696f84a15fb3e37/apps/zotonic_mod_zotonic_site_management/src/zotonic_status_addsite.erl#L316) the files are just copied.
Maybe could the `copy_site_file` function be renamed to `copy_file_replacing_tags`, `copy_erl_file` or something like that for a better understanding.

Reply:
Yep, all is ok here.